src/go/nerdctl-stub/parse_args.go

```diff
@@ -278,6 +278,13 @@ func init() {
 	registerArgHandler("compose", "-f", filePathArgHandler)
 	registerArgHandler("compose", "--project-directory", filePathArgHandler)
 	registerArgHandler("compose", "--env-file", filePathArgHandler)
+	subcommands := []string{"build", "config", "down", "kill", "logs", "ps", "pull", "push", "up"}
+	for _, sc := range subcommands {
```

Comment:
```suggestion
	for _, sc := range []string{"build", "config", "down", "kill", "logs", "ps", "pull", "push", "up"} {
```
We never use the slice for anything else.

Reply:
sure